packages/chat-component-bindings/src/utils/updateMessagesWithAttached.test.ts

```diff
@@ -56,4 +64,25 @@ describe('update message with attached status', () => {
     updateMessagesWithAttached(messagesArrayWithOtherMessage);
     expect(getAttachedStatusArray(messagesArrayWithOtherMessage)).toEqual([false, undefined, false]);
   });
+
+  test('Set right status for attached property for messages with more than or equal to 5 mins time gap', () => {
+    const currentDateTime = new Date();
+    const messagesArrayWithOtherMessage = [
+      cannedCustomDateTimeChatMessage('1', currentDateTime),
+      cannedCustomDateTimeChatMessage('1', new Date(currentDateTime.getDate() + 5 * 60000)),
```

Comment:
Could also test mins like 2 or 4 to see if that works well

Reply:
updated